src/component/utils/isEventHandled.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule isEventHandled
+ * @typechecks
+ * @flow
+ */
+
+'use strict';
+
+import type {DraftHandleValue} from 'DraftHandleValue';
+
+/**
+ * Utility method for determining whether or not the value returned
+ * from a handler indicates that it was handled.
+ */
+function isEventHandled(value: DraftHandleValue): boolean {
+  return value === 'handled';
```

Comment:
For (temporary) backwards compatibility, this should probably also deal with `value === true`. Then we can cut the cord in a subsequent breaking change.